src/library_glfw.js

```diff
@@ -1228,7 +1228,8 @@ var LibraryGLFW = {
   glfwGetMonitors: function(count) {
     setValue(count, 1, 'i32');
     if (!GLFW.monitors) {
-      GLFW.monitors = allocate([1, 0, 0, 0], 'i32', ALLOC_NORMAL);
+      GLFW.monitors = _malloc(Runtime.POINTER_SIZE);
+      setValue(GLFW.monitors, 1, 'i32');
```

Comment:
better to use `{{{ makeSetValue }}}` here.

Reply:
This particular file seems to use `setValue` everywhere..  so better to be locally consistent and maybe mass-convert this file at some point?

Reply:
Oh, ok, that does sound better.